deblur/workflow.py

```diff
@@ -131,34 +152,59 @@ def remove_artifacts_seqs(seqs_fp,
         number of threads to use for SortMeRNA
     verbose: boolean, optional
         If true, output SortMeRNA errors
+    sim_thresh: float, optional
```

Comment:
Missing the description of `coverage_thresh`.

Also, in this case, instead of giving the default as 0, is better to have `None` so the description is something like:
`If not provided, 0.65 is used if negate=False and 0.95 for negate=True`.